src/plugins/saved_objects_management/public/management_section/recent_work.tsx

```diff
@@ -102,14 +107,18 @@ export const RecentWork = (props: { core: CoreStart; workspaceEnabled?: boolean
     const options: string[] = [allOption];
     detailedSavedObjects
       .filter((item) => !item.error)
-      .forEach((recentAccessItem: ChromeRecentlyAccessedHistoryItem) => {
-        if (recentAccessItem.meta?.type && options.indexOf(recentAccessItem.meta.type) === -1) {
-          options.push(recentAccessItem.meta.type);
+      .forEach((recentAccessItem) => {
+        if (recentAccessItem?.type && options.indexOf(recentAccessItem?.type) === -1) {
+          options.push(recentAccessItem?.type);
         }
       });
     return options.map((option: string) => ({ label: option, value: option }));
   }, [detailedSavedObjects]);
 
+  const capitalTheFirstLetter = function (recentAccessItem: DetailedRecentlyAccessedItem) {
+    return recentAccessItem.type.charAt(0).toUpperCase() + recentAccessItem.type.slice(1);
+  };
+
```

Comment:
Nit: seems like an util function, better to move it out of the Component render function.